### PR TITLE
Replace curl with wget, remove apt-mark showauto for Plone 5

### PR DESCRIPTION
**Pull request by svx**
This PR does the following:

- Replaces ``curl`` with ``wget``, since ``wget`` has less dependencies
- Removes apt-mark showauto

### CODE REVIEW

**Comment by svx** [comment]
Jenkins sucks, the Changelog **is** updated !

**Comment by gforcada** [comment]
@svx that would be @mister-roboto not checking for ``CHANGES.md``. I will update it.

**Comment by gforcada** [comment]
*Side note*: No need for *sucks* though...

**Comment by svx** [comment]
Apologies, let me rephrase:

Jenkins has currently a disagreement with me, cause of a not known file-extension, with the result that a test is failing and both of us are not happy.

**Comment by gforcada** [comment]
@svx no problem, it was an easy fix fortunately 😄
